.gitignore

```diff
@@ -30,3 +30,6 @@ yarn-error.log*
 
 # web-hint report
 hint-report/
+
+#Redis dump file
+dump.rdb
```

Comment:
I'm confused as to why you're adding this.  Why do we want this included?  You're not using it anywhere.  Also, why would you add this to the `.gitignore` and then add it in your commit?

Reply:
I added it into my previous commit, when I noticed it I realized that we dont need that at all, since it is generated by Redis, so I added that to .gitignore.

Reply:
That's fine (i.e., adding to `.gitignore`).  But you also added it to this PR (the file itself):

<img width="1026" alt="Screen Shot 2019-11-29 at 9 15 28 PM" src="https://user-images.githubusercontent.com/427398/69894408-6b999d80-12ed-11ea-9585-eec79bc00814.png">